tools/lkl/include/lkl.h

```diff
@@ -35,15 +35,13 @@ extern "C" {
 int inet_pton(int af, const char *src, void *dst);
 #endif
 
-#if __LKL__BITS_PER_LONG == 32 && (defined(__ANDROID__) || defined(__i386__))
-#define __lkl__NR_fcntl __lkl__NR_fcntl64
```

Comment:
I guess android-aarch64 case should use this block of ifdef ?

Reply:
No, that's not it, because that block wasn't used on aarch64 Android.